src/biomappings/resources/mappings.tsv

```diff
@@ -8967,8 +8972,10 @@ mondo	0010693	nystagmus 1, congenital, X-linked	skos:exactMatch	mesh	C537853	Nys
 mondo	0010717	pyruvate dehydrogenase E1-alpha deficiency	skos:exactMatch	mesh	C564071	Pyruvate Dehydrogenase E1 Alpha Deficiency	semapv:ManualMappingCuration	orcid:0000-0001-9439-5346			
 mondo	0010743	thrombocytopenia 1	skos:exactMatch	mesh	C564052	Thrombocytopenia 1	semapv:ManualMappingCuration	orcid:0000-0001-9439-5346			
 mondo	0010760	XH antigen	skos:exactMatch	mesh	C009691	Xh antigen	semapv:ManualMappingCuration	orcid:0000-0001-9439-5346			
+mondo	0010778	cyclic vomiting syndrome	skos:exactMatch	mesh	C536228	Familial cyclic vomiting syndrome	semapv:ManualMappingCuration	orcid:0000-0002-1216-4761	semapv:LexicalMatching	generate_mondo_mesh_mappings.py	0.9
```

Comment:
it's not clear from the MONDO entry if these are an exact match, or if a "familial" variant is a narrower match

Reply:
@cthoyt in this case although the OMIM entry is mentioned in both Mondo and Mesh, but because of the lack of clarity it would be advisable to put this in the unsure group.